### PR TITLE
add team retrospectives (points that were voted for)

### PR DESCRIPTION
**Pull request by jsms90**
To replace the `retrospectives` repo in the FAC10 organisation

### CODE REVIEW

File: stop-go-continue/fac-10/team-week-1.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Week 1
+
+## PAMF Action Points
+
+### Stop
++ Merging own pair's pull requests
+
+### Go
++ More code reviews - explaining code to each other through the day
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-1.md]: 👍

File: stop-go-continue/fac-10/team-week-1.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
++ Adding to readme as we go through
+
+## Facalacalaca Action Points
+
+### Stop
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-1.md]: 👍

File: stop-go-continue/fac-10/team-week-1.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
++ Comparing yourself other teams
+
+### Go
++ Deliver a comprehensive README at the end of the project
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-1.md]: 👍

File: stop-go-continue/fac-10/team-week-1.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
++ Internal team code reviews
++ Complimenting & motivating each other
+
+## JZLP Action Points
+
+### Go
++ Split tasks into components so that everyone can work on tasks simultaneously
++ Focus on user-stories
++ Time presentations so that everyone has an equal time slot
++ Comment code and ensure that everyone understands the code 100%
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-1.md]: 👍

File: stop-go-continue/fac-10/team-week-1.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## FACX Machine
+
+### Stop
++ Blindly merging PRs
+
+### Go
++ Real pairing
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-1.md]: As opposed to?

File: stop-go-continue/fac-10/team-week-3.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+### Stop
+N/A
+
+### Go
++ BEM
++ Divide work equally
+
+### Continue
++ Laughter
```
Comment (iteles) [inline stop-go-continue/fac-10/team-week-3.md]: <3

**Comment by bradreeder** [comment]
Lack of issue! @jsms90 😋 

But looks good!

**Pull request merged**